cloudant-client/src/main/java/com/cloudant/client/api/model/DesignDocument.java

```diff
@@ -446,6 +457,20 @@ public String getDbCopy() {
             return dbcopy;
         }
 
+        /**
+         * @param element JSON object or string
+         * @return a string form of the object or the JSON string with leading and trailing "
+         * stripped
+         */
+        private String stringifyMRElement(JsonElement element) {
+            String s = element.toString();
```

Comment:
Yeah there should be a `null` check, thanks. I've added it in 198ed8b.
I also reversed the checking to use `isJsonString` instead of `isJsonObject` so that we only attempt to strip quotes if we are sure it is a string, as other JSON elements would not be quoted. Even though it (at least currently) should only be an Object or a String it should be less fragile this way.
@rhyshort FYI in case you wanted to look at this change and change your +1